text/chapter5.md

```diff
@@ -171,9 +178,7 @@ We are able to append additional fields to the record, and the `showPerson` func
 Type of expression lacks required label "last"
 ```
 
-We can read the new type signature of `showPerson` as "takes any record with `first` and `last` fields which are `Strings` _and any other fields_, and returns a `String`".
-
-This function is polymorphic in the _row_ `r` of record fields, hence the name _row polymorphism_.
+We can read the new type signature of `showPerson` as "takes any record with `first` and `last` fields which are `Strings` _and any other fields_, and returns a `String`". This function is polymorphic in the _row_ `r` of record fields, hence the name _row polymorphism_.  Note that this behavior is different than that of the original `showPerson`. Without the row variable `r`, `showPerson` only accepts records with _exactly_ a `first` and `last` field and no others. 
```

Comment:
This seemed worth making a special note of to me for the sake of anybody coming from a JS or TS background, where this is actually unintuitive (though probably quite intuitive for anybody coming from a Haskell background).  In TS, for example, you can fully type annotate everything but your functions will still always accept records that contain a superset of fields defined in the type signature.  See [this](https://stackoverflow.com/questions/49580725/is-it-possible-to-restrict-typescript-object-to-contain-only-properties-defined)